website/docs/theming/basic.mdx

```diff
@@ -200,3 +200,69 @@ future.
 
 > All Chakra's component follows this API and we've made call component styles
 > available to you in the `chakra/` folder of your project.
+
```

Comment:
I moved the Badge example from the overview to here. Having it on the overview felt like a deep dive too quickly while ignoring a basic intro on how theming locally/globally works. It feels redundant because the Button example is probably enough... but I think you can never have too many examples.

Reply:
I think this change makes sense, especially in combination with the changes introducing global theming in the overview.